apps/web/src/modules/job/list/pages/hooks/useMobileStyle/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
```

Comment:
consider using `useSwitch` (my bad, I forget to write it in change log)

Reply:
oh, ok